main.tf

```diff
@@ -0,0 +1,119 @@
+locals {
+  app_name_and_env = "${var.app_name}-${data.terraform_remote_state.common.outputs.app_env}"
+  app_env          = data.terraform_remote_state.common.outputs.app_env
```

Comment:
I kind of wonder whether we should deprecate `app_env` and only use the longer version. It seems a little silly to have both.